components/layout_2020/style_ext.rs

```diff
@@ -77,6 +78,19 @@ impl ComputedValuesExt for ComputedValues {
         matches!(size, Size::Auto)
     }
 
+    fn inline_box_offsets_are_both_auto(&self) -> bool {
+        let position = self.get_position();
+        let offsets = if self.writing_mode_is_horizontal() {
+            (position.left, position.right)
+        } else {
+            (position.top, position.bottom)
+        };
+        matches!(
+            offsets,
+            (LengthPercentageOrAuto::Auto, LengthPercentageOrAuto::Auto)
+        )
```

Comment:
Not sure why you always use `matches!` where a simple equality is enough. :)

Reply:
Out of habit from some enums that don’t implement `PartialEq`. But yeah `==` would be better.